cmd/dlv/dlv_test.go

```diff
@@ -180,17 +180,17 @@ func testOutput(t *testing.T, dlvbin, output string, delveCmds []string) (stdout
 
 	// ignore "dlv debug" command error, it returns
 	// errors even after successful debug session.
-	cmd.Wait()
+	_ = cmd.Wait()
 	stdout, stderr = stdoutBuf.Bytes(), stderrBuf.Bytes()
 
 	_, err = os.Stat(debugbin)
 	if err == nil {
 		// Sometimes delve on Windows can't remove the built binary before
 		// exiting and gets an "Access is denied" error when trying.
-		// See: https://travis-ci.com/go-delve/delve/jobs/296325131)
+		// See: https://travis-ci.com/go-delve/delve/jobs/296325131.
 		// We have added a delay to gobuild.Remove, but to avoid any test
 		// flakiness, we guard against this failure here as well.
-		if runtime.GOOS != "windows" || !strings.Contains(err.Error(), "Access is denied") {
+		if runtime.GOOS != "windows" {
```

Comment:
But we do want to complain for errors other than access is denied.

Reply:
The enclosing branch checks that `err == nil`.

Reply:
Ah ok, you are right.